<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-KF67JSQ" failures="3" tests="4" name="ebanking app(failed)" time="23.335" errors="0" timestamp="2021-03-18T19:28:32 PDT">
  <testcase classname="com.inetbanking.testcases.TC_1LoginDDT" name="LoginDDT" time="3.185">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testcases.TC_1LoginDDT.LoginDDT(TC_1LoginDDT.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
  <testcase classname="com.inetbanking.testcases.TC_1LoginDDT" name="LoginDDT" time="2.638">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testcases.TC_1LoginDDT.LoginDDT(TC_1LoginDDT.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
  <testcase classname="com.inetbanking.testcases.TC_1LoginDDT" name="LoginDDT" time="5.375"/>
  <testcase classname="com.inetbanking.testcases.TC_1LoginDDT" name="LoginDDT" time="2.525">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testcases.TC_1LoginDDT.LoginDDT(TC_1LoginDDT.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginDDT -->
</testsuite> <!-- ebanking app(failed) -->
